nemo_skills/training/openrlhf/llm_reward.py

```diff
@@ -1,21 +1,40 @@
 import torch
-from nemo_skills.inference.server.model import get_model
-from nemo_skills.prompt.utils import get_prompt
+
 from nemo_skills.code_execution.math_grader import extract_answer
 from nemo_skills.evaluation.metrics.utils import is_correct_judgement
+from nemo_skills.inference.server.model import get_model
+from nemo_skills.prompt.utils import get_prompt
+from nemo_skills.utils import prefill_judgement
 
 
 def reward_func(queries: list[str], prompts: list[str], prompt_metadata: list[dict]):
```

Comment:
A one-line description that the function assigns a binary score to the prompt-response/problem-response pairs.